tests/ttnn/unit_tests/operations/test_full.py

```diff
@@ -0,0 +1,102 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+import torch
+import torch.nn as nn
+import ttnn
+from models.utility_functions import comp_allclose
+from loguru import logger
+
+from tests.ttnn.utils_for_testing import assert_equal
+
+
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [32, 32],  # single tile
+        # [5, 17, 31],  # multiple tiles
+    ],
+)
```

Comment:
Is there any reason you commented this out? I think you should add more test cases.

Reply:
There might be some mistakes in the rebase process. I've uncommented it